apps/src/sites/code.org/pages/public/teacher-dashboard/sections.js

```diff
@@ -33,9 +31,6 @@ export function renderSectionsPage(data) {
   store.dispatch(setStudioUrl(data.studiourlprefix));
   store.dispatch(setValidLoginTypes(data.valid_login_types));
   store.dispatch(setValidGrades(data.valid_grades));
-  if (!experiments.isEnabled('sectionFocus')) {
-    store.dispatch(setValidAssignments(data.valid_courses, data.valid_scripts));
```

Comment:
Actually, we cant remove the pegasus stuff since we still need to check if a course is valid when assigning it :(